src/pages/Admin/AdminEvent.tsx

```diff
@@ -85,7 +85,7 @@ const AdminEvent: React.FunctionComponent<PropsType> = ({
         price: 0,
         collective_price: 0,
         host: data.host,
-        room: data.room,
+        room: data.room ?? "",
```

Comment:
Can you not send the field at all instead?

Reply:
Yes okay